minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -241,14 +241,8 @@ protected void setupTest() throws IOException {
     // If we are using the LocalMiniDFSCluster or S3UnderStorageCluster or OSSUnderStorageCluster,
     // we need to update the UNDERFS_ADDRESS to point to the cluster's current address.
     // This must happen after UFS is started with UnderFileSystemCluster.get().
-    // TODO(andrew): Move logic to the alluxio-tests module so that we can use instanceof here
-    // instead of comparing classnames.
-    if (mUfsCluster.getClass().getSimpleName().equals("LocalMiniDFSCluster")
-        || mUfsCluster.getClass().getSimpleName().equals("S3UnderStorageCluster")
-        || mUfsCluster.getClass().getSimpleName().equals("OSSUnderStorageCluster")) {
-      String ufsAddress = mUfsCluster.getUnderFilesystemAddress() + mHome;
-      Configuration.set(Constants.UNDERFS_ADDRESS, ufsAddress);
-    }
+    String ufsAddress = mUfsCluster.getUnderFilesystemAddress() + mHome;
+    Configuration.set(Constants.UNDERFS_ADDRESS, ufsAddress);
```

Comment:
The ufs address was set as "${alluxii.home}/underFSStorage" before it is set to a new one. The default was pick up from the common resources properties file.

The new address is set to something like s3://bucket/var/folders/..../test-cluster; where s3://bucket/ is initialized after the first call to UnderStorageCluster.get; it reads the value based on test profile. The behavior is different for LocalFileSystemCluster.